Makefile

```diff
@@ -142,14 +142,15 @@ check-deps:
 DOCKER_USERNAME?=jujusolutions
 JUJUD_STAGING_DIR=/tmp/jujud-operator
 JUJUD_BIN_DIR=${GOPATH}/bin
+OPERATOR_IMAGE_TAG = $(shell juju version | cut -d- -f1,2)
```

Comment:
Given that you are making the image by copying in jujud, shouldn't you use jujud for the version?

Reply:
Can do. On a build system, juju and jujud will return the same thing, but yeah, makes sense to change.